wasp/apps/flashlight.py

```diff
@@ -21,19 +20,31 @@ class TorchApp(object):
 
     def foreground(self):
         """Activate the application."""
-        self.draw()
         wasp.system.request_tick(1000)
+        wasp.system.request_event(wasp.EventMask.TOUCH)
 
         self._brightness = wasp.system.brightness
         wasp.system.brightness = 3
```

Comment:
I think you need to restore the brightness from the last time the app was opened here